src/status_im/contexts/onboarding/create_password/view.cljs

```diff
@@ -58,16 +41,21 @@
                            (not passwords-match?)
                            (not empty-password?))]
     [:<>
-     [password-with-hint
-      {:hint           {:text   (i18n/label :t/password-creation-hint)
-                        :status hint-1-status
-                        :shown  true}
+     [password-with-hint/view
+      {:hint           (if (not password-short-enough?)
+                         {:text   (i18n/label
+                                   :t/password-creation-max-length-hint)
+                          :status :error
+                          :shown? true}
+                         {:text   (i18n/label :t/password-creation-hint)
+                          :status hint-1-status
+                          :shown? true})
```

Comment:
Here we're showing the "maximum 100 characters" error for the Create Password form.